enclave/crypto/mbedtls/init.c

```diff
@@ -24,3 +30,21 @@ int oe_is_symcrypt_engine_available()
 {
     return 0;
 }
+
+int oe_is_symcrypt_provider_available()
+{
+    return 0;
+}
+
+int OSSL_provider_init(
+    const void* handle,
+    const void* in,
+    const void** out,
+    void** provctx)
+{
+    OE_UNUSED(handle);
+    OE_UNUSED(in);
+    OE_UNUSED(out);
+    OE_UNUSED(provctx);
+    return 0;
+}
```

Comment:
Clang probably produces a warning for this, though it should've been fixed by clang-format. Could you please add a newline at EOF?

Reply:
add newline